### PR TITLE
Made template for User Guide and added most of baseline infos

### PR DESCRIPTION
**Pull request by vishnuvk47**
--> also changed the code that removes leading zeros to be after the .trim

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,42 +1,212 @@
...
+
+Examples:
+```
+switch-to jenson
+```
+
+
+### Add a savings Goal to the current account: `save`
+
+Adds a savings goal to teh current account. <br />
```
Comment (tyuyang) [inline docs/UserGuide.md:121]: teh to the

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,42 +1,212 @@
...
+will raise an alert when attempting to withdraw more than the savings goal, if attempt is before deadline
```
Comment (tyuyang) [inline docs/UserGuide.md:122]: will to Will.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,42 +1,212 @@
...
+
+Format: `save AMOUNT` >> will then prompt the user for the deadline.
+
+* The amount to save has to be a number, -ve numbers alloed -- can be used to overwrite as no savings goal
```
Comment (tyuyang) [inline docs/UserGuide.md:126]: alloed to allowed

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,42 +1,212 @@
...
+* deadline entered has to be in dd-mm-YYYY format. No other format will be entertained by the program
```
Comment (tyuyang) [inline docs/UserGuide.md:127]: deadline to Deadline

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1,42 +1,212 @@
...
+* any error will only be displayed after the deadline is received and processed.
```
Comment (tyuyang) [inline docs/UserGuide.md:128]: any to Any

File: docs/UserGuide.md [hunk 6]
```diff
@@ -1,42 +1,212 @@
...
+
+Examples:
+```
+save 300
+>>What would be the end date for the duration of your Saving goal?
+11-11-2011
+>>try saving a Minimum of $300 until 11-11-2011
```
Comment (tyuyang) [inline docs/UserGuide.md:135]: Wait this is really what is printed in the program? Cos try should be Try and Minimum shld be minimum

**Review by tyuyang — APPROVED** [review]
Just commented on your part since you might as well finalise your part of the UG in this pr. You might also want to include the view-account command since the feature is also attributed to you. The main bulk of the content is fine for a template

**Comment by vishnuvk47** [comment]
Have resolved the issue where 0.10 would be saved as .10 when trying to implement a method to avoid leading zeroes. Now program would accurately display the number in its most valid state. Have also corrected for the Minor Spelling/typos.

**Pull request merged**